mne_bids/copyfiles.py

```diff
@@ -533,18 +533,19 @@ def copyfile_eeglab(src, dest):
     # structure and potentially breaks re-reading of the file
     uint16_codec = None
     eeg = loadmat(file_name=src, simplify_cells=False,
-                  appendmat=False, uint16_codec=uint16_codec)
+                  appendmat=False, uint16_codec=uint16_codec, mat_dtype=True)
     oldstyle = False
     if 'EEG' in eeg:
         eeg = eeg['EEG']
         oldstyle = True
 
+    has_fdt_link = False
```

Comment:
I would suggest to revert this change, as it makes the code flow less obvious

Reply:
In the current implementation, the is some potential cases where has_fdt_link will not be set and a KeyError thrown.
It happened to me during testing so that's how I tried to fixed that. I can revert that change still.

Reply:
For me both ways are fine. Thanks for figuring out that there is one potential flow that previously lead to an error!